src/mono/mono.proj

```diff
@@ -368,8 +347,8 @@ JS_ENGINES = [NODE_JS]
       Condition="'$(ActualWasiSdkVersion)' != '$(ExpectedWasiSdkVersion)'" />
 
     <!-- LLVM in WASI SDK 22 will call wasm-opt when found on the PATH. But it will fail because wasm-opt can't read the WASM components.
-    After we upgrade to WASI SDK 23, we could use no-wasm-opt LLVM option to avoid this issue.
-    See https://github.com/llvm/llvm-project/pull/95208#issuecomment-2220400454
+    After we upgrade to WASI SDK 23.x, we could use no-wasm-opt LLVM option to avoid this issue.
```

Comment:
Is this comment still relevant?

Reply:
I believe that it will land with LLVM 19, we are still on LLVM 18.
I will update the comment in another PR and I updated https://github.com/dotnet/runtime/issues/104773